run_all_unit_tests.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 source ~/.bashrc
 
+CUDA_HOME=/opt/share/cuda-11.8 #to prevent mismatch between cuda runtime and the cuda version used to compile pytorch
```

Comment:
Maybe the path should be in envs.sh or in Jenkins file?

Reply:
Good question, not sure